packages/Collapsible/src/Collapsible.styles.js

```diff
@@ -17,23 +17,38 @@ const collapsibleStyles = css`
   .collapsible__label {
     display: inline-block;
     border-radius: ${tokens.border.radius};
-    width: 100%;
+    margin-left: -${tokens.spaceSm};
+    margin-right: -${tokens.spaceSm};
+    padding-left: ${tokens.spaceSm};
+    padding-right: ${tokens.spaceSm};
+    width: calc(100% + ${tokens.space});
+  }
+
+  .collapsible__icon--left {
+    margin-right: ${tokens.spaceSm};
   }
 
+  .collapsible__icon--right {
+    float: right;
+    margin-left: ${tokens.spaceSm};
+  }
+  
   .collapsible__label--is-toggle-icon-only {
+    margin: 0;
     width: auto;
   }
 
-  [data-pka-anchor="collapsible.icon"] svg {
-    vertical-align: -0.1em;
+  .collapsible__label--left {
+    margin-right: ${tokens.spaceSm}; 
   }
 
-  .collapsible--icon-left {
-    padding-right: ${tokens.spaceSm};
+  .collapsible__label--right {
+    float: right;
+    margin-left: ${tokens.spaceSm};
   }
 
-  .collapsible--icon-right {
-    float: right;
+  [data-pka-anchor="collapsible.icon"] svg {
+    vertical-align: -0.1em;
```

Comment:
`em`s? Thought we weren't using 'em?

Reply:
Generally, yeah, we aren't... and we definitely don't for `font-sizes` (for now, though we probably _should_ use `rems` for a11y) but for properties that affect the vertical alignment of elements to text, I think it still makes sense to use a unit that is relative to the size of that text. I think they are cleaner than doing some math like:
https://github.com/acl-services/workpapers/pull/12414/files#diff-9e9d9ccf6f85f99f09849f0d8ad08db4R19

Reply:
I don't see the benefit... in case the font-size of the component changes, you wont have to update the vertical-align as well?

Oh man i'm glad I didnt approve that other PR